docker-compose.override.dev.yml

```diff
@@ -5,6 +5,12 @@ networks:
     name: moneywise
 
 services:
+  moneywise-ui:
+    environment: 
+      - REACT_APP_MONEYWISE_BASE_URL=http://localhost:8000
```

Comment:
Again, `environment` is not needed for moneywise UI. It's only needed at build time, which is what build args is doing. Can remove this section.

Reply:
Actually, even better: removing this. Having a UI container locally has not been helpful, has actually hurt my ability to start coding.

Reply:
Done: 8d0a29ee6760924964d7bf92acaa4b3ac6079303